tests/awards/manage-applications/awards-page-title.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { Page } from 'playwright';
+import BrowserService from '../../../src/services/utils/browser.service';
+import { ManageApplications } from '../../../src/pages/awards/manage-applications.page';
```

Comment:
Absolute paths are currently not supported in `ts-node`. https://github.com/wclr/ts-node-dev/issues/95

I tried some of the solutions mentioned, but its not working and I have already spent a lot of time in this. Lets leave this for now, I will keep trying.

Reply:
Don't worry about this, but if you want to check this in the future it seems this can work:
https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353

I have also created a package.json file in the `src` folder and then reference it in the root `package.json`. Like this:

```js
// /package.json
{
  "dependencies": {
    "cases-e2e": "./src"
  }
}
```

```js
// /src/pacakge.json
{
  "name": "cases-e2e"
}
```

Then on any TS file I can do this:

```ts
import something from 'cases-e2e/a/path/from/src/something';
```